<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-TPTMM17A" failures="1" tests="1" name="Testing Element - Delete" time="9.05" errors="0" timestamp="2022-09-26T21:07:27 IDT">
  <testcase classname="addressbook.DeleteElementTask" name="testViewTask" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0">
      <![CDATA[java.lang.RuntimeException: java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
at java.base/java.util.Objects.checkIndex(Objects.java:359)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at addressbook.DeleteElementTask.viewTask(DeleteElementTask.java:38)
... 29 more
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testViewTask -->
</testsuite> <!-- Testing Element - Delete -->
